lib/src/models/reaction_popup_configuration.dart

```diff
@@ -49,7 +51,11 @@ class ReactionPopupConfiguration {
   /// Provides glass morphism effect configuration.
   final GlassMorphismConfiguration? glassMorphismConfig;
 
+  /// Provides callback when user react on message.
+  final void Function(Message message, String emoji)? onUserReacted;
```

Comment:
can we change name of this from `onUserReacted` to `userReactionCallback` ?

Reply:
Resolved

Reply:
not reflected yet.

Reply:
Resolved